services/spar/src/Spar/Options.hs

```diff
@@ -44,6 +48,20 @@ deriveOpts raw = do
         derivedOptsBindCookiePath = URI.uriPath respuri
         unwrap = maybe (throwM $ ErrorCall "Bad server config: no domain in response URI") pure
     derivedOptsBindCookieDomain <- URI.hostBS . URI.authorityHost <$> unwrap (URI.uriAuthority respuri)
+
+    -- We could also make this selectable in the config file, but it seems easier to derive it from
+    -- the SAML base uri.
+    let derivedOptsScimBaseURI = (saml raw ^. SAML.cfgSPSsoURI) & pathL %~ derive
+          where
+            derive path = case reverse .
+                               filter (not . SBS.null) .
+                               SBS.split (ascii '/') $
```

Comment:
Dependency to `Data.ByteString`, ie. package `bytestring`?  i doubt spar will build without that.  :-)

Am I missing anything?

Reply:
Dependency on `text-latin1`, where `ascii` comes from.

On the other hand apparently we're bound to depend on it anyway (because `data-timeout` does), so whatever.